.kokoro/lint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
```

Comment:
Do these files need to be ${currentYear} when generated?

Reply:
the `.kokoro` ones we decided no, because they're all identical from a template and that code was written in 2018 (it would be a pain to have all our templates be updated each year for each repo).